applications/notify/src/notify_fax_outbound_to_email.erl

```diff
@@ -115,7 +115,7 @@ create_template_props(Event, [FaxDoc | _Others]=_Docs, Account) ->
                  ,{<<"call_id">>, kz_json:get_value(<<"Call-ID">>, Event)}
                   | fax_values(kz_json:get_value(<<"Fax-Info">>, Event))
                  ]}
-    ,{<<"account_db">>, kz_doc:account_db(Account)}
+    ,{<<"account_db">>, kapi_fax:account_db(Event)}
```

Comment:
@mark2600 this works because `kapi_fax:account_db` looks for `Account-DB` in the payload, but the payload sent is actually a `notify`. maybe add the same lookup to `kapi_notifications` ?